17:55:09,420 ERROR main DijkstraAlgorithm:execute:32 - RuntimeException in DijkstraAlgorithm.getDistance
